<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Timing</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Train delay</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScienDSpinBox" name="trainDelaySpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Train duration</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScienDSpinBox" name="trainDurationSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Phase 1 duration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ScienDSpinBox" name="phase1DurationSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="phase2DurationLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Phase 2 duration</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ScienDSpinBox" name="phase2DurationSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="interPhaseIntervalLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Interphase interval</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ScienDSpinBox" name="interPhaseIntervalSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pulse Interval</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ScienDSpinBox" name="pulseIntervalSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="burstModeGroupBox">
     <property name="title">
      <string>Burst mode</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Burst duration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScienDSpinBox" name="burstDurationSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Interburst interval</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScienDSpinBox" name="interBurstIntervalSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>3600.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="3" column="0">
       <widget class="QPushButton" name="softTriggerPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Trigger !</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/lightning.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="trigger2CheckBox">
        <property name="text">
         <string>Trigger 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="trigger1CheckBox">
        <property name="text">
         <string>Trigger 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="amplitudesGroupBox">
     <property name="title">
      <string>Amplitudes</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>defines the voltage between phases, pulses and pulse trains</string>
        </property>
        <property name="text">
         <string>Baseline voltage</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScienDSpinBox" name="baselineVoltageSpinBox">
        <property name="toolTip">
         <string>defines the voltage between phases, pulses and pulse trains</string>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Phase 1 voltage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScienDSpinBox" name="phase1VoltageSpinBox">
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="phase2VoltageLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Phase 2 voltage</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ScienDSpinBox" name="phase2VoltageSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>-5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="customTrainGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;NOT YET IMPLEMENTED&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Sends a sequence of onset times and voltages describing a train of pulses. Two custom trains can be created, with 1,000 pulses each.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Custom pulse train</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Custom Train</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="customTrainSourceComboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Train #1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Train #2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Pulses</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">customTrainButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Bursts</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">customTrainButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="customTrainLoopCheckBox">
        <property name="text">
         <string>Loop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="customTrainLoadPushButton">
        <property name="text">
         <string>Load...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="outputModeGroupBox">
     <property name="title">
      <string>Output mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="outputModeMonophasicRadioButton">
        <property name="text">
         <string>Monophasic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">outputModeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outputModeBiphasicRadioButton">
        <property name="text">
         <string>Biphasic</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">outputModeButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="schemaLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>310</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>310</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>GUI.scientific_spinbox</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="outputModeButtonGroup"/>
  <buttongroup name="customTrainButtonGroup"/>
 </buttongroups>
</ui>
